### PR TITLE
Switch to latest edtt

### PR DESCRIPTION
**Pull request by erbr-ot**
This will switch to use the most recent version of EDTTool.

Also introduces a couple of fixups/commits re. EDTT app.

Note that the west.yml temporarily refers to EDTTool/EDTT.git repo. Once the review looks good, the mirror will be updated as will the reference in west.yml. This update will also change to use branch 'main' as default on EDTT

### CODE REVIEW

**Comment by wopu-ot** [comment]
I think commit "edtt: make sure the refactored LLCP is correctly selected" should go into a separate PR to separate concerns.

**Comment by erbr-ot** [comment]
I agree, re. 8b62442654fbce487f2936e97f086a4308c7551d - this is already on its way into main through other channels, but I wanted to make this Draft PR 'self-contained'

**Review by kruithofa — DISMISSED** [review]

File: west.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: edttool
```
Comment (wopu-ot) [inline west.yml:24]: The main branch of this repository is now mirrored, so this can be remoed.

Comment (erbr-ot) [reply west.yml]: Excellent.

**Comment by erbr-ot** [comment]
Updated edtt reference in west.yml and rebased.

**Review by wopu-ot — APPROVED** [review]

**Review by cvinayak — APPROVED** [review]
quick ack

**Review by carlescufi — APPROVED** [review]

**Pull request merged**